harness/tests/.gitignore

```diff
@@ -3,3 +3,5 @@
 
 # Ignore Gradle build output directory
 build
+
+graal/godot-kotlin-graal-jni-config.json
```

Comment:
Why should we ignore this?

Reply:
We ignore it because gradle plugin add it if it is not here.